torch_xla/experimental/pallas_kernels/ragged_paged_attention_v2.py

```diff
@@ -151,9 +151,6 @@ def check_inputs_shapes(
     raise ValueError(
         f"Expected {cu_q_lens.shape=} to be ({max_num_seqs + 1},)  where"
         " `max_num_seqs` is `page_indices.shape[0]`.")
-  if max_num_seqs > max_num_batched_tokens:
```

Comment:
ooc, why is it removed?

Reply:
The check might fail. BTW, max_num_batched_tokens is not a appropriate name here.